libraries/AP_HAL_SITL/SITL_State.h

```diff
@@ -90,14 +172,15 @@ class HALSITL::SITL_State {
     uint16_t current2_pin_value;  // pin 14
 
     // paths for UART devices
-    const char *_uart_path[7] {
+    const char *_uart_path[8] {
         "tcp:0:wait",
         "GPS1",
         "tcp:2",
         "tcp:3",
         "GPS2",
         "tcp:5",
-        "tcp:6",
```

Comment:
Any reason not to push the number of uart paths out further?

Reply:
We are limited to the number SerialManager slots. We can increase that if you want. But with existing use case it doesn't feel necessary.